### DISCUSSION TITLE
sendPacket doesn't work

### DISCUSSION THREAD

**Discussion by smpak19** [ANLAB-KAIST/KENSv3#114] [Project Q&A 2. Transport Layer (TCP)] [answered]
When I call this->sendPacket(), wireshark captures it well. But I couldn't receive any packetArrived callbacks.
I checked that the packet arrives well to host, but it lost somewhere in ipv4->tcp module (sendPacket in ipv4.cpp not work).
Is it my implementation problem? or something related to my OS because me and my teammate both use MacOS.
I was trying to handle this more than 2 days. :(
Thank you.

**Comment by smpak19** [comment, marked as answer]
[Solved] It's just our implementation mistake :)

**Reply by harryoh99** [reply]
Can you briefly give me a hint of what the problem was?? :( Thanks
